apps/ensindexer/Dockerfile

```diff
@@ -1,27 +1,17 @@
-# Base working stage
-FROM node:18-slim
-ARG PNPM_VERSION=9.12.0
-RUN apt-get update && \
-    apt-get install -y \
-        curl \
-        gnupg \
-        python3 \
-        make \
-        g++ && \
```

Comment:
I wonder if we might face [`node-gyp` build issues](https://namehash.slack.com/archives/C086Z6FNBHN/p1742206103516189?thread_ts=1742136784.637199&cid=C086Z6FNBHN) while installing npm deps?